components/prize_pool/commons/prize_pool_base.lua

```diff
@@ -484,10 +484,25 @@ function BasePrizePool._comparePrizes(x, y)
 	return sortX == sortY and x.index < y.index or sortX < sortY
 end
 
+function BasePrizePool:_shouldDisplayPrizeSummary()
+	-- if prizeSummary is disabled do not show it
+	if not self.options.prizeSummary then
+		return false
+	end
+
+	local baseMoney = tonumber(Variables.varDefault('tournament_prizepool' .. BASE_CURRENCY:lower())) or 0
```

Comment:
```suggestion
	local baseMoney = tonumber(Variables.varDefault('tournament_prizepool' .. BASE_CURRENCY:lower()))
```
should we only not display it if the value is 0
i.e. display it for unset/empty value?
(if 0 is entered in infobox the var holds 0 if 0 is entered as (usd) prize pool in infobox league)